examples/hello-compute/main.rs

```diff
@@ -25,12 +25,8 @@ fn main() {
         limits: wgpu::Limits::default(),
     });
 
-    let cs_bytes = include_bytes!("shader.comp.spv");
-    let mut cs_words = vec!();
-    for bytes4 in cs_bytes.chunks(4) {
-        cs_words.push(u32::from_le_bytes([bytes4[0], bytes4[1], bytes4[2], bytes4[3]]));
-    }
-    let cs_module = device.create_shader_module(&cs_words);
+    let cs = include_bytes!("shader.comp.spv");
+    let cs_module = device.create_shader_module(&wgpu::read_spirv(std::io::Cursor::new(&cs[..])).unwrap());
```

Comment:
we should be able to use `cs` instead of `&cs[..]` here and in other places

Reply:
```
   Compiling wgpu v0.2.2 (/home/rubic/Projects/Crates/wgpu/wgpu-rs)
error[E0277]: the trait bound `[u8; 1576]: std::convert::AsRef<[u8]>` is not satisfied
  --> examples/hello-compute/main.rs:29:50
   |
29 |     let cs_module = device.create_shader_module(&wgpu::read_spirv(std::io::Cursor::new(cs)).unwrap());
   |                                                  ^^^^^^^^^^^^^^^^ the trait `std::convert::AsRef<[u8]>` is not implemented for `[u8; 1576]`
   |
   = help: the following implementations were found:
             <[T; 0] as std::convert::AsRef<[T]>>
             <[T; 10] as std::convert::AsRef<[T]>>
             <[T; 11] as std::convert::AsRef<[T]>>
             <[T; 12] as std::convert::AsRef<[T]>>
           and 30 others
   = note: required because of the requirements on the impl of `std::convert::AsRef<[u8]>` for `&[u8; 1576]`
   = note: required because of the requirements on the impl of `std::io::Read` for `std::io::Cursor<&[u8; 1576]>`
   = note: required by `gfx_hal::pso::read_spirv`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0277`.
error: Could not compile `wgpu`.
```